rest-service/manager_rest/resources_v2.py

```diff
@@ -960,9 +962,18 @@ def list_events(self, query, include=None):
     def _search(self, query, include=None):
         es = ManagerElasticsearch.get_connection()
 
-        return es.search(index=self._set_index_name(),
-                         body=query,
-                         _source=include or True)
+        try:
+            return es.search(index=self._set_index_name(),
+                             body=query,
+                             _source=include or True)
+        except elasticsearch.TransportError as e:
+            # This happens when the events index is in the process
+            # of being created.
+            if (e.status_code == 503 and
+                    'SearchPhaseExecutionException' in e.error):
+                return {'hits': {'hits': [], 'total': 0}}
```

Comment:
for the record i dont really like this ("hits" etc.) being here - totally not the place for it, but seeing as how the whole implementation of _search doesnt really belong in this module we might as well :/